controllers/syncer/csi_node.go

```diff
@@ -74,20 +78,20 @@ func NewCSINodeSyncer(c client.Client, scheme *runtime.Scheme, driver *csiscaleo
 	}
 
 	return syncer.NewObjectSyncer(config.CSINode.String(), driver.Unwrap(), obj, c, func() error {
-		return sync.SyncFn(daemonSetRestartedKey, daemonSetRestartedValue)
+		return sync.SyncCSIDaemonsetFn(daemonSetRestartedKey, daemonSetRestartedValue)
 	})
-	//return "hello"
 }
 
-func (s *csiNodeSyncer) SyncFn(daemonSetRestartedKey string, daemonSetRestartedValue string) error {
-	logger := csiLog.WithName("SyncFn")
+// SyncCSIDaemonsetFn handles reconscilation of CSI driver daemonset.
+func (s *csiNodeSyncer) SyncCSIDaemonsetFn(daemonSetRestartedKey string, daemonSetRestartedValue string) error {
+	logger := csiLog.WithName("SyncCSIDaemonsetFn")
 
 	out := s.obj.(*appsv1.DaemonSet)
 
 	secrets := []corev1.LocalObjectReference{}
 	if len(s.driver.Spec.ImagePullSecrets) > 0 {
 		for _, s := range s.driver.Spec.ImagePullSecrets {
-			logger.Info("SyncFn: Got ", "ImagePullSecret:", s)
+			logger.Info("SyncCSIDaemonsetFn: Got ", "ImagePullSecret:", s)
```

Comment:
We can remove "SyncCSIDaemonsetFn" in the log as we have set "logger := csiLog.WithName("SyncCSIDaemonsetFn")".  So this will take care of logging in this function with "SyncCSIDaemonsetFn" name